web/js/deerInitializer.js

```diff
@@ -119,14 +133,22 @@ DEER.TEMPLATES.objectsAsDropdown = function(obj, options = {}) {
  */
 DEER.TEMPLATES.locationsMulti = function(obj, options = {}) {
     try {
+        let whichCollection = UTILS.getLabel(obj) ? UTILS.getLabel(obj) : ""
+        let quickAddTmpl = `<a title="Click here to add a new entity by name." class="tag bg-primary text-white is-small pull-right" onclick="LR.ui.toggleEntityAddition(event, this.nextElementSibling)">&#x2b;</a>

```

Comment:
Copy-Paste` warning. You will be very sad later if you don't resolve this soon.

Reply:
I KNOW ALRIGHT I just wanted it to work.  The whole things isn't optimal, I don't think the different `multi`s need to be their own thing.  This is just filling in while I think about it some more.